includes/export/abstract-wc-csv-batch-exporter.php

```diff
@@ -55,11 +56,11 @@ protected function get_file_path() {
 	 */
 	public function get_file() {
 		$file = '';
-		if ( @file_exists( $this->get_file_path() ) ) {
-			$file = @file_get_contents( $this->get_file_path() );
+		if ( @file_exists( $this->get_file_path() ) ) { // phpcs:ignore Generic.PHP.NoSilencedErrors.Discouraged
+			$file = @file_get_contents( $this->get_file_path() ); // phpcs:ignore Generic.PHP.NoSilencedErrors.Discouraged, WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents, WordPress.WP.AlternativeFunctions.file_system_read_file_get_contents
```

Comment:
Should we maybe consider using the alternative function instead of ignoring the PHPCS rule? I haven't checked the details but just checking to make sure we verify this before merging.